packages/itwinui-css/backstop/engine_scripts/puppet/onReady.js

```diff
@@ -2,9 +2,14 @@ module.exports = async (page, scenario, vp) => {
   console.log('SCENARIO > ' + scenario.label);
   await require('./overrideCss')(page, scenario);
 
-  if (scenario.label.startsWith('dark_')) {
+  if (scenario.label.endsWith('_light')) {
+    await page.evaluate(() => (document.documentElement.dataset.iuiTheme = 'light'));
+  } else if (scenario.label.endsWith('_dark')) {
     await page.evaluate(() => (document.documentElement.dataset.iuiTheme = 'dark'));
   }
+  if (scenario.label.includes('_hc_')) {
```

Comment:
To make it safer, you could check of `_hc_light` or `_hc_dark` to avoid catching `_hc_` that's part of the real label
```suggestion
  if (scenario.label.endsWith('_hc_light') || scenario.label.endsWith('_hc_dark')) {
```

Reply:
_hc_ should not be part of the label because everything with underscores is something we add to the name.

Reply:
I'm slightly confused by what you meant. But to confirm, should it be `scenario.label.endsWith` or `scenario.label.includes`? [L10](https://github.com/iTwin/iTwinUI/pull/822/commits/e3298df92fdc7b99b28b3137792da6e0b7f2c42c#)

Since I think the labels always have the theme information at the end, maybe `endsWith` is a better check.

Reply:
Oh, yeah, I can update that.

Reply:
Is it good now?

Reply:
Yes, thanks